pkg/sql/update.go

```diff
@@ -86,13 +88,21 @@ func (r *updateRun) initRowContainer(params runParams, columns colinfo.ResultCol
 	for i := range r.resultRowBuffer {
 		r.resultRowBuffer[i] = tree.DNull
 	}
+
+	if ots := params.extendedEvalCtx.SessionData().OriginTimestampForLogicalDataReplication; ots.IsSet() {
+		r.originTimestampCPutHelper.OriginTimestamp = ots
+	}
 }
 
 func (u *updateNode) startExec(params runParams) error {
 	// cache traceKV during execution, to avoid re-evaluating it for every row.
 	u.run.traceKV = params.p.ExtendedEvalContext().Tracing.KVTracingEnabled()
 
-	u.run.initRowContainer(params, u.columns)
+	u.run.init(params, u.columns)
+
+	if ots := params.extendedEvalCtx.SessionData().OriginTimestampForLogicalDataReplication; ots.IsSet() {
```

Comment:
is this required, given the init call above?

Reply:
🤦 no. I added it to init then forgot to clean up the other initialization points.